handlers/accesscontrol/access-control.go

```diff
@@ -236,6 +282,12 @@ func (ac *AccessControlHandlersCollection) cachePlaybackAccessControlInfo(playba
 		}
 	}
 
+	if refreshInterval > 0 {
+		if _, ok := refreshIntervalCache.data[playbackID]; !ok {
+			refreshIntervalCache.data[playbackID] = &RefreshIntervalRecord{RefreshInterval: refreshInterval, LastRefresh: time.Now()}
```

Comment:
do we need to lock when updating this map? since it's being read in the 5 second sleep loop in another routine